src/components/Property/RaiseWorkOrder/RateScheduleItemView.js

```diff
@@ -27,6 +28,7 @@ const RateScheduleItemView = ({
     []
   )
   const [rateScheduleItemCosts, setRateScheduleItemCosts] = useState([])
+  const ADDING_MULTIPLE_SOR_PAGE = 2
```

Comment:
Assigning to the same const in multiple places as we have here and in the parent `RaiseWorkOrderFormView.js` can lead us to tricky bugs later on. One const value could be renamed or reassigned but without remembering to update the other. Suggest we instead pass down a function which manages the page transition in reference to the const and give it a specific name like `changePageToAddMultipleSORs` or similar. This also helps readability when reading the parent - it's clear that the child can change to a specific page, but not any page.